### ISSUE TITLE
api.downloadVideoByIdNoWatermark is not working

### ISSUE THREAD

**Issue by DangoLee** [avilash/TikTokAPI-Python#27]
Hello,
I‘m trying to downloadVideoByIdNoWatermark, it returns a html code with error instead.

    def downloadVideoByIdNoWatermark(self, video_id, save_path):
        video_info = self.getVideoById(video_id)
        video_url = video_info["itemInfo"]["itemStruct"]["video"]["downloadAddr"]
        video_data = get_req_text(video_url, params=None, headers=self.headers)
        pos = video_data.find("vid:")
        if pos == -1:
            raise VideoException("Video without watermark not available in new videos")
        video_url_no_wm = "https://api2-16-h2.musical.ly/aweme/v1/play/?video_id={" \
                          "}&vr_type=0&is_play_url=1&source=PackSourceEnum_PUBLISH&media_type=4" \
            .format(video_data[pos + 4:pos + 36])
        print("video_url_no_wm : ")
        print(video_url_no_wm)
        video_data_no_wm = get_req_content(video_url_no_wm, params=None, headers=self.headers)
        print("video_data_no_wm : ")
        print(video_data_no_wm)
        with open(save_path, 'wb') as f:
            f.write(video_data_no_wm)

video_id:
6829613641491418374

video_url_no_wm :
https://api2-16-h2.musical.ly/aweme/v1/play/?video_id=v09044b80000br3qikhe3ejq8khkj6n0&vr_type=0&is_play_url=1&source=PackSourceEnum_PUBLISH&media_type=4

video_data_no_wm :
***
b'<HTML><HEAD>\n<TITLE>Access Denied</TITLE>\n</HEAD><BODY>\n<H1>Access Denied</H1>\n \nYou don\'t have permission to access "&#47;D&#47;11965&#47;819927&#47;000&#47;api2&#46;musical&#46;ly&#47;c2f7c0a5a85fc47409196837f0d0d4d3&#47;6011a5d1&#47;video&#47;tos&#47;useast2a&#47;tos&#45;useast2a&#45;pve&#45;0068&#47;3cb3c791b9c048caaed792fae13091cc&#47;&#63;a&#61;1233&#38;br&#61;8404&#38;bt&#61;4202&#38;cd&#61;0&#37;7C0&#37;7C0&#38;cr&#61;0&#38;cs&#61;0&#38;cv&#61;1&#38;dr&#61;0&#38;ds&#61;6&#38;er&#61;&#38;l&#61;202101271141260101901760164F090CD3&#38;lr&#61;&#38;mime&#95;type&#61;video&#95;mp4&#38;pl&#61;0&#38;qs&#61;0&#38;rc&#61;MzlubnRoaG50dTMzOzczM0ApaTQzPGU7ZWVnN2g3aWc5OGdrZ2U1ZzBlZjBfLS1fMTZzc18tXjIvYGM1MmIxYl5gX2I6Yw&#37;3D&#37;3D&#38;vl&#61;&#38;vr&#61;" on this server.<P>\nReference&#32;&#35;18&#46;f23ca17&#46;1611747688&#46;193a3168\n</BODY>\n</HTML>\n'
***

**Comment by avilash** [comment]
The old method to download videos without watermark no longer works, and I'm yet to figure out the new way.
As of now you cannot download videos without watermark, using the package. I should have updated it in the Readme

There are websites online which allow you to download without watermark, so there should be a way

**Comment by DangoLee** [comment]
Thanks for your reply.

**Comment by flashclub** [comment]
> The old method to download videos without watermark no longer works, and I'm yet to figure out the new way. As of now you cannot download videos without watermark, using the package. I should have updated it in the Readme
> 
> There are websites online which allow you to download without watermark, so there should be a way

Hi, thank you very much for writing this open source project. I was very excited when I first found this project, I simply used the method downloadVideoById and it made it easy to download the video, then I saw that the video had a watermark on it, so I tried again with downloadVideoByIdNoWatermark and it reported an error, then I turned to this issue. considering your previous reply was January 2021, are there any plans to perfect the watermark-free downloads now?
In any case, thank you very much.
